src/styles/main.scss

```
@@ -1,3 +1,69 @@
-body {
-  margin: 0;
+@import './block/varibles.scss';
+
+.center-block {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;;
+  flex-wrap: wrap;
+  justify-content: left;
+  padding: $padding;
+  gap: $gap;
+  width: $square-size * $columns + ($gap * $columns - 1);
+
+  &__day {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    box-sizing: border-box;
+    border: 1px solid black;
+    text-align: center;
+    background-color: $main-color;
+
+    height: $square-size;
+    width: $square-size;
+
+    font-size: $number-size;
+    font-family: $number-font;
+
+    cursor: pointer;
+    transition: background-color 0.5s, transform 0.5s;
```

Create variable for transition duration 
```suggestion
    transition: background-color ..., transform ...;
```